requirements.txt

```diff
@@ -1 +1,2 @@
 SQLAlchemy
+redis-py-cluster>=2.1.0
```

Comment:
`git+https://github.com/Grokzen/redis-py-cluster.git@master`
pypi is still on `2.0.0`, which will not play well with redis-py 3.5.3.

Reply:
Since dependencies version are not yet officially available I had marked it as draft version.
But if you need you may setup offline packages from source code.

Check
https://github.com/Grokzen/redis-py-cluster/blob/2.0.99/setup.py
Here version is mentioned as 2.1.0 that's why mentioned required version as 2.1.0. 
Old version of redis-py-cluster is not compatible with redis-py version.

Reply:
`2.1.0` is not yet released to pypi, but this is about to change really soon.